Formula/p/pythran.rb

```diff
@@ -38,6 +37,11 @@ class Pythran < Formula
     sha256 "9c270fe5f4b130969b54174de7db4e764b09b4f7f67ccfc32480e29f78348d97"
   end
 
+  resource "setuptools" do
+    url "https://files.pythonhosted.org/packages/fc/c9/b146ca195403e0182a374e0ea4dbc69136bad3cd55bc293df496d625d0f7/setuptools-69.0.3.tar.gz"
+    sha256 "be1af57fc409f93647f2e8e4573a142ed38724b8cdd389706a867bb4efcf1e78"
+  end
```

Comment:
This is causing conflict with `depends_on "python-setuptools"`.

Due to https://github.com/Homebrew/brew/pull/16442, `setuptools` may need to be specified in 
https://github.com/Homebrew/homebrew-core/blob/c68391c81ea022b47cbe006baa8b7252873eb332/pypi_formula_mappings.json#L997-L998

Reply:
I just removed `python-setuptools` formula dependency now. relates to https://github.com/Homebrew/homebrew-core/pull/159725

Reply:
Thanks, the part that had me confused was whether I still needed distutils as a depends_on, but also to exclude it in that pypi mappings file. Will confess that I don't understand much about how Brew's python distribution/packaging is wrapped at an internal level!